common/src/main/java/org/tron/common/parameter/CommonParameter.java

```diff
@@ -169,6 +170,8 @@ public class CommonParameter {
   @Setter
   public int minParticipationRate;
   @Getter
+  public P2pConfig p2pConfig;
+  @Getter
```

Comment:
This modification has nothing to do with this PR, right?

Reply:
Yes, there is something wrong when split pr. I have deleted it.